db/TableSchemasV2.sql

```diff
@@ -0,0 +1,113 @@
+-- Set up table and autoincrementing primary key
+CREATE TABLE Image_Info (
+    ImageId SERIAL PRIMARY KEY,
+    OriginalImageName text NOT NULL,
+    ImageLocation text,
+    Height integer NOT NULL,
+    Width integer NOT NULL,
+    ModifiedDtim timestamp NOT NULL default current_timestamp,
+    CreatedDtim timestamp NOT NULL default current_timestamp
+);
+
+-- Set up table
+CREATE TABLE TagState (
+    TagStateId integer NOT NULL,
+    TagStateName text NOT NULL
+);
+
+-- Set up table
+CREATE TABLE Image_Tags (
+    ImageTagId integer NOT NULL,
+    ImageId integer REFERENCES Image_Info(ImageId) ON DELETE RESTRICT,
+    Class text NOT NULL, --Needed?
```

Comment:
Not sure if this schema handles this scenario, let me know if this is already addressed or any thoughts you have. Sorry, should've looked into this earlier.

```
"1012.png": [
            {
                "x1": 31.54617414248021,
                "y1": 302.3115577889447,
                "x2": 68.88654353562005,
                "y2": 331.8994974874372,
                "width": 488,
                "height": 512,
                "box": {
                    "x1": 31.54617414248021,
                    "y1": 302.3115577889447,
                    "x2": 68.88654353562005,
                    "y2": 331.8994974874372
                },
                "UID": "7c25a40e",
                "id": 0,
                "type": "Rectangle",
                "tags": [
                    "class1",
                    "class2"
                ],
                "name": 1
            },
            {
                "x1": 392.0738786279683,
                "y1": 343.47738693467335,
                "x2": 428.12664907651714,
                "y2": 383.356783919598,
                "width": 488,
                "height": 512,
                "box": {
                    "x1": 392.0738786279683,
                    "y1": 343.47738693467335,
                    "x2": 428.12664907651714,
                    "y2": 383.356783919598
                },
                "UID": "f2373f90",
                "id": 1,
                "type": "Rectangle",
                "tags": [
                    "class1",
                    "class2",
                    "class3"
                ],
                "name": 2
            }
        ]
```

for this one image, it has two "tags", and in each tag, they have multiple "classes" assigned to them. In this case, would we need another table?

Reply:
@mtarng The Image_Tags table is supposed to handle this but currently it does not reflect the entire corpus of the JSON VOTT. I would like to learn more about the magnitude of the tags attribute (the 3 classes). How often will this be accessed? 

Keep in mind I expect the VOTT json to change over time. I purposely did not reach full fidelity with the VOTT json this week